e2e/node/e2e.test.ts

```diff
@@ -95,21 +105,24 @@ const invalidCredentialClaims = {
   },
 };
 
-const env = getNodeTestingEnvironment({
+const {
+  idp: oidcIssuer,
+  vcProvider,
+  clientCredentials,
+} = getNodeTestingEnvironment({
```

Comment:
I thought we agreed on not destructuring this?

Reply:
https://github.com/inrupt/solid-client-vc-js/pull/531/commits/f9acfd45b54ae92f31ede14e7e270d90d73e4bbc